package.json

```diff
@@ -18,6 +18,9 @@
     "svelte",
     "README.md"
   ],
+  "engines": {
+    "node": "^8"
```

Comment:
`^8` is equivalent to `>= 8.0.0 < 9.0.0`, that means Svelte doesn't support Node.js 10 and 12. I assume it's not intended.

Reply:
d'oh